src/IdentityManager/Directory.Build.props

```diff
@@ -0,0 +1,25 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Authors>ServiceTitanCamp Identity team</Authors>
+    <Company>ServiceTitanCamp</Company>
+    <Description>IdentityManager for travely project</Description>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+  </PropertyGroup>
+  <PropertyGroup>
+    <SolutionDir Condition="$(SolutionDir) == ''">$(MSBuildThisFileDirectory.TrimEnd('\').TrimEnd('/'))\</SolutionDir>
+    <ArtifactsDir>$(SolutionDir)artifacts</ArtifactsDir>
```

Comment:
I case of this repo, we can also add service name in output path. So we can introduce ServiceName variable and add it to aritfacts path.
```
<ServiceName>identitymanager</ServiceName>
 <ArtifactsDir>$(SolutionDir)artifacts/$(ServiceName)</ArtifactsDir>
```